addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml

```diff
@@ -5,7 +5,14 @@
             <t t-if="messageInReplyToView">
                 <t t-if="!messageInReplyToView.messageView.message.parentMessage.isEmpty">
                     <b class="o_MessageInReplyToView_author text-muted ml-2">@<t t-esc="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
-                    <span class="o_MessageInReplyToView_body ml-1" t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody" t-on-click="messageInReplyToView.onClickReply"/>
+                    <t t-if="!messageInReplyToView.messageView.message.parentMessage.isBodyEmpty">
+                        <span class="o_MessageInReplyToView_body ml-1" t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody" t-on-click="messageInReplyToView.onClickReply"/>
+                    </t>
+                    <t t-if="messageInReplyToView.messageView.message.parentMessage.isBodyEmpty and messageInReplyToView.messageView.message.parentMessage.hasAttachments">
```

Comment:
condition too long on template, I'm suggesting to create a new computed field in messageInReplyToView to hold this value

Reply:
The line that is too long is caused by improper read in the chain `messageInReplyToView.messageView` is wrong, it should have been immediately `messageInReplyToView.message`.

And I'm not sure it worth making a computed field for this expression. It will be shorter, but you'll struggle finding a good and brief name for this field, which you be a smell that it's an inappropriate attempt to define a concept on this expression.